src/index.html

```diff
@@ -13,20 +17,27 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./styles/style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <!-- #endregion -->
+  <body class="body page__body">
+    <form class="form body__form">
+      <input
+        type="text"
+        placeholder="Try “Los Angeles”"
+        data-qa="big"
+        class="form__search form__search--big"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="form body__form">
```

Comment:
There should be a 20px spacing between the two search bars as per the task requirements. Consider adding a CSS rule to achieve this spacing.